komodo/symlink/create_links.py

```diff
@@ -17,6 +17,13 @@ def working_dir(path):
         os.chdir(prev_dir)
 
 
+def traverse_symlink_tree(key, link_dict):
+    values = [k for k, v in link_dict.items() if v == key]
+    for value in values:
+        traverse_symlink_tree(value, link_dict)
+        print(f"{value} was implicitly moved")
```

Comment:
This means we will get one line of output for every level in the link-tree. Will that lead to too much noise in the slack message? Perhaps we only want the topmost change, or it might be possible to give all the information in one line.

Reply:
I don't know how long a symlink-chain usually is, but it would be better to know too much than not enough. I will try out different formattings, maybe we can shorten it without leaving out information.